### ISSUE TITLE
Release plan - v23.x Current

### ISSUE THREAD

**Issue by RafaelGSS** [nodejs/Release#1045]
_Draft schedule - all dates subject to change_

Version | Release Date | Releaser
--------|--------------| -------------
v23.0.0 |  2024-10-16  | @RafaelGSS
v23.1.0 |  2024-10-24  | @aduh95
v23.2.0 |  2024-11-11  | @aduh95
v23.3.0 |  2024-11-20  | @RafaelGSS 
v23.4.0 |  2024-12-10  | @targos and @aduh95 
v23.5.0 |  2024-12-19  | @aduh95 
v23.6.0 |  2024-01-07  | @marco-ippolito 
v23.6.1 (security) |  2025-01-21  | @RafaelGSS 
v23.7.0 |  2025-01-30  | @aduh95
v23.8.0 |  2025-02-13  | @targos
v23.9.0 |  2025-02-26  | @targos
v23.10.0 |  2025-03-13  | @aduh95
v23.11.0 |  2025-04-01 | @aduh95
(Maintenance transition) |  2025-04-01  | _No release_
v23.11.1 (security) |  2025-05-14  | @RafaelGSS

**Comment by RafaelGSS** [comment]
Yes, I am. I'm just trying to solve https://github.com/nodejs/node/pull/55690 to use it

**Comment by marco-ippolito** [comment]
If you don't mind I'd like to do the next 23 release. v23.5.0 needs to be updated

**Comment by MikeMcC399** [comment]
@RafaelGSS 

Will somebody be updating https://github.com/nodejs/Release/blob/main/README.md to move Node.js `23.x` to https://github.com/nodejs/Release#end-of-life-releases now that the EOL date 2025-06-01 has passed?
